src/internal/FluencePeer.ts

```diff
@@ -0,0 +1,350 @@
+import { AirInterpreter, CallServiceResult, LogLevel, ParticleHandler, SecurityTetraplet } from '@fluencelabs/avm';
+import log from 'loglevel';
+import { Multiaddr as MA } from 'multiaddr';
```

Comment:
Let's keep `Multiaddress`, `MA` is cryptic

Reply:
ok